packages/react/src/provider.tsx

```diff
@@ -3,14 +3,15 @@ import { Client, OpenFeature } from '@openfeature/web-sdk';
 
 type ProviderProps = {
   client?: Client;
+  name?: string,
```

Comment:
Allow the provider to be scoped by name.

Reply:
Could you add a JSDoc to both properties?

Reply:
Done